### PR TITLE
Requirements: update `newrelic`

### PR DESCRIPTION
**Pull request by humitos**
I tested it on one web and one build-default instance and it worked fine. Hopefully, it doesn't start logging all DEBUG messages again when deployed in production for real :)

Closes: https://github.com/readthedocs/readthedocs-ops/issues/1290
Requires: https://github.com/readthedocs/readthedocs-ops/pull/1376

### CODE REVIEW

**Comment by ericholscher** [comment]
Yea, seems like this is a wonky upgrade :(

**Comment by humitos** [comment]
I merged https://github.com/readthedocs/readthedocs.org/pull/10544 into this PR to be able to run `inv requirements.update`

**Comment by humitos** [comment]
I solved the problem of the DEBUG log lines in https://github.com/readthedocs/readthedocs-ops/pull/1376. I tested this in production and it worked great. We have an instance using the latest New Relic agent now and it is not logging garbage 👍🏼

**Comment by humitos** [comment]
This PR is ready to be merged.

File: requirements/pip.in [hunk 1]
```diff
@@ -33,8 +33,7 @@ jsonfield
 requests
 requests-toolbelt
 slumber
-# awscli (requirements/docker.txt) isn't compatible with pyyaml >= 5.5.
```
Comment (ericholscher) [inline requirements/pip.in:36]: Nice, this got fixed as well?

**Review by ericholscher — APPROVED** [review]

File: requirements/pip.in (hunk 1 above)
Comment (ericholscher) [reply requirements/pip.in:36]: Found the answer: https://github.com/readthedocs/readthedocs.org/pull/10544 :)

**Pull request merged**